### PR TITLE
Refcount

### PR DESCRIPTION
**Pull request by btsimonh**
Adds Matrix.getrefCount()
Only really useful for debugging, but not harmful
consider addref() to be detrimental - dangerous if someone used it by mistake.

### CODE REVIEW

**Comment by btsimonh** [comment]
Please don't merge!
Further investigation shows that mat.release() decrements the ref count, and then ditches the link to the data, regardless of the ref count.  So addref really does not actually do anything we could want? (I think).
reading the ref count is interesting from a debug perspective....  but more thought required.

**Comment by btsimonh** [comment]
Peter - I leave this one to you to decide.  It's not harmful in itself, but only useful if debugging.

File: src/Matrix.h [hunk 1]
```diff
@@ -134,7 +134,10 @@ class Matrix: public node_opencv::Matrix{
   JSFUNC(Shift)
   JSFUNC(Reshape)
 
+          
+  JSFUNC(Addref)
```
Comment (peterbraden) [inline src/Matrix.h]: We should probably kill this one too then.

**Comment by btsimonh** [comment]
good spot.  done..

**Comment by peterbraden** [comment]
Thanks!

**Pull request merged**